Px.Utils/Validation/SyntaxValidation/KeyValuePairValidationEntry.cs

```diff
@@ -0,0 +1,40 @@
+﻿using PxUtils.PxFile;
+
+namespace PxUtils.Validation.SyntaxValidation
+{
+    /// <summary>
+    /// Represents a key-value pair validation entry. A key-value pair validation entry is a type of validation entry that contains a key-value pair. This class implements the <see cref="IValidationEntry"/> interface.
```

Comment:
Maybe "Represents a key value pair of a px file metadata entry. For example KEY[en]="value" where key is .. and value is ..